e2e/js/js-esm/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "outDir": "./dist/",
+      "noImplicitAny": true,
+      "module": "es6",
+      "target": "es2015",
+      "jsx": "react",
+      "allowJs": false,
+      "skipLibCheck": true,
+      "experimentalDecorators": true,
+      "moduleResolution": "Node"
```

Comment:
why use Node here?

Reply:
renamed it to `node`, just a holdover from the test app i ripped this from

Reply:
it shouldn't be node either, that option is basically deprecated

Reply:
you probably want "bundler"
https://www.typescriptlang.org/tsconfig#moduleResolution

Reply:
`bundler` gives me an error running webpack:

``TS6046: Argument for '--moduleResolution' option must be: 'node', 'classic', 'node16', 'nodenext'.```

Reply:
hm that's weird, wonder if the version of TS or webpack is too old

Reply:
yeah actually it's cause of the old typescript package being included, i'll fix it